src/azure-cli/azure/cli/command_modules/container/_help.py

```diff
@@ -106,3 +106,40 @@
     text: az container show --name MyContainerGroup --resource-group MyResourceGroup
     crafted: true
 """
+
+helps['container container-group-profile'] = """
+type: group
+short-summary: Manage Azure Container Instance Container Group Profile.
+"""
+
+helps['container container-group-profile create'] = """
+type: command
+short-summary: Create a container group profile.
+examples:
+  - name: Creates a container group profile.
```

Comment:
please use first-person verb for all help messages.
```suggestion
  - name: Create a container group profile.
```